diff --git a/machine_code_final/machine_code_final.ino b/machine_code_final/machine_code_final.ino
index 4bb5a44..1a3a664 100644
--- a/machine_code_final/machine_code_final.ino
+++ b/machine_code_final/machine_code_final.ino
@@ -1,12 +1,12 @@
 // home btn vars
-int home_btn = 2; // 2
+int home_btn = 10; // 2
 bool home_btn_pressing = false;
 int home_btn_pressed = false;
 unsigned long home_btn_millis = 0;
 unsigned long home_btn_delay = 500;
 
 // feedback btn vars
-int feedback_btn = 4; // 2
+int feedback_btn = 11; // 2
 bool feedback_btn_pressing = false;
 bool stage_one_feedback_btn_pressed = false;
 bool stage_two_feedback_btn_pressed = false;
@@ -20,7 +20,7 @@ unsigned long homing_millis = 0;
 unsigned long homing_delay = 5000;
 
 // start btn vars
-int start_btn = 3; // 3
+int start_btn = 8; // 3
 bool start_btn_pressing = false;
 int start_btn_pressed = false;
 unsigned long start_btn_millis = 0;
@@ -53,13 +53,13 @@ unsigned long trigger_two_delay = 700;
 unsigned long currentMillis = 0;
 bool debug = true;
 bool debug_delay = false;
-int trigger = 10; // 10
+int trigger = 12; // 10
 
 void setup() {
   Serial.begin(115200);
-  pinMode(home_btn, INPUT_PULLUP);
-  pinMode(start_btn, INPUT_PULLUP);
-  pinMode(feedback_btn, INPUT_PULLUP);
+  pinMode(home_btn, INPUT);
+  pinMode(start_btn, INPUT);
+  pinMode(feedback_btn, INPUT);
 
   //pinMode(the_led, OUTPUT);
   pinMode(trigger, OUTPUT);
@@ -73,7 +73,7 @@ void loop() {
 
   // home button rutine
 
-  if (digitalRead(home_btn) == 0 && home_btn_pressing == false && home_btn_pressed == false) {
+  if (digitalRead(home_btn) == 1 && home_btn_pressing == false && home_btn_pressed == false) {
     home_btn_pressing = true;
     if (debug == true) {
       Serial.println("home btn pressing ...");
@@ -94,7 +94,7 @@ void loop() {
       Serial.println(currentMillis - home_btn_millis);
     }
   }
-  if (digitalRead(home_btn) == 1 && home_btn_pressing == true && home_btn_pressed == false) {
+  if (digitalRead(home_btn) == 0 && home_btn_pressing == true && home_btn_pressed == false) {
     home_btn_pressing = false;
     if (debug == true) {
       Serial.println("home btn not pressing ...");
@@ -129,7 +129,7 @@ void loop() {
 
   // start button rutine
 
-  if (digitalRead(start_btn) == 0 && start_btn_pressing == false && start_btn_pressed == false && mech_homed == true) {
+  if (digitalRead(start_btn) == 1 && start_btn_pressing == false && start_btn_pressed == false && mech_homed == true) {
     start_btn_pressing = true;
     if (debug == true) {
       Serial.println("start btn pressing ...");
@@ -150,7 +150,7 @@ void loop() {
       Serial.println(currentMillis - start_btn_millis);
     }
   }
-  if (digitalRead(start_btn) == 1 && start_btn_pressing == true && start_btn_pressed == false) {
+  if (digitalRead(start_btn) == 0 && start_btn_pressing == true && start_btn_pressed == false) {
     start_btn_pressing = false;
     if (debug == true) {
       Serial.println("start btn not pressing ...");
@@ -203,7 +203,7 @@ void loop() {
     }
   }
 
-  if (digitalRead(feedback_btn) == 0 && trigger_one_over == true && feedback_btn_pressing == false && stage_one_feedback_btn_pressed == false) {
+  if (digitalRead(feedback_btn) == 1 && trigger_one_over == true && feedback_btn_pressing == false && stage_one_feedback_btn_pressed == false) {
     feedback_btn_pressing = true;
     feedback_btn_millis = currentMillis;
     //Serial.println("feedback....");
@@ -213,7 +213,7 @@ void loop() {
     feedback_btn_pressing = false;
     Serial.println("stage one feedback....");
   }
-  if (digitalRead(feedback_btn) == 1 && feedback_btn_pressing == true && stage_one_feedback_btn_pressed == false) {
+  if (digitalRead(feedback_btn) == 0 && feedback_btn_pressing == true && stage_one_feedback_btn_pressed == false) {
     feedback_btn_pressing = false;
   }
 
@@ -271,7 +271,7 @@ void loop() {
     }
   }
 
-  if (digitalRead(feedback_btn) == 0 && trigger_two_over == true && feedback_btn_pressing == false && stage_two_feedback_btn_pressed == false) {
+  if (digitalRead(feedback_btn) == 1 && trigger_two_over == true && feedback_btn_pressing == false && stage_two_feedback_btn_pressed == false) {
     feedback_btn_pressing = true;
     feedback_btn_millis = currentMillis;
     //Serial.println("feedback....");
@@ -281,7 +281,7 @@ void loop() {
     feedback_btn_pressing = false;
     Serial.println("stage two feedback....");
   }
-  if (digitalRead(feedback_btn) == 1 && feedback_btn_pressing == true && stage_two_feedback_btn_pressed == false) {
+  if (digitalRead(feedback_btn) == 0 && feedback_btn_pressing == true && stage_two_feedback_btn_pressed == false) {
     feedback_btn_pressing = false;
   }
 
